shared/teams/channel/tabs/index.tsx

```diff
@@ -0,0 +1,123 @@
+import * as React from 'react'
+import * as Types from '../../../constants/types/teams'
+import * as ChatTypes from '../../../constants/types/chat2'
+import * as Kb from '../../../common-adapters'
+import flags from '../../../util/feature-flags'
+import {
```

Comment:
I know a lot of this is copy-pasted from the main teams stuff, but since you're touching it anyway it might be nice to clean it up a bit - can you change this to `import * as Styles` to be consistent with how we do stuff nowadays?